mkosi/__init__.py

```diff
@@ -3345,6 +3354,8 @@ def build_stuff(uid: int, gid: int, config: MkosiConfig) -> None:
             if p.name.startswith(state.config.output.name):
                 compress_output(state.config, p, uid=state.uid, gid=state.gid)
 
+    print_output_size(config)
```

Comment:
That's a bit of a non-topical addition here?

Reply:
It's required to make this work actually, we don't have permission to read certain directories outside of the user namespace, so calculating the size of the output only works inside the image if we're outputting a directory.